includes/paragraphs.inc

```diff
@@ -72,30 +75,19 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_text_feature_media(array &$
   /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
   $paragraph = $variables['paragraph'];
 
-  // Link field doesn't exist in oe_paragraphs <= 1.8.0, so ensure it exists.
-  if ($paragraph->hasField('field_oe_link') && !$paragraph->get('field_oe_link')->isEmpty()) {
```

Comment:
link is not needed ?

Reply:
We have a problem with the cyclomatic complexity threshold and I had to remove as many "if" as possible. Link already comes in content so for this reason I don't need to add to variables